package.json

```diff
@@ -1,51 +1,43 @@
 {
-  "name": "dcos-ui-kit",
+  "name": "@dcos/ui-kit",
   "version": "0.1.0",
   "description": "DCOS UI Kit",
   "scripts": {
-    "preinstall": "./scripts/pre-install",
-    "start": "NODE_ENV=development npm run storybook --root ./ ",
-    "dist": "npm run clean && NODE_ENV=production webpack -p --config ./webpack.config.js",
-    "lint": "tslint -c tslint.json --project tsconfig.json",
-    "test": "jest --config ./jest.config.json",
-    "test:watch": "jest --watch",
-    "commitlint": "commitlint -e",
-    "storybook": "start-storybook -p 6006",
     "build:storybook": "build-storybook",
+    "clean": "rimraf dist",
+    "commitlint": "commitlint -e",
+    "create:component": "./scripts/create-component",
     "deploy:storybook": "storybook-to-ghpages",
-    "clean": "rm -rf dist",
+    "dist": "tsc -p tsconfig.dist.json",
+    "predist": "npm run clean",
+    "lint": "tslint -c tslint.json --project tsconfig.json",
+    "prerelease": "npm run dist",
     "release": "standard-version",
-    "create:component": "./scripts/create-component"
+    "setup:git-hooks": "./scripts/pre-install",
```

Comment:
why was this moved from automatic setup `preinstall` to a custom script `setup:git-hooks`?

Reply:
As you might have seen already in the commit message of commit f76bb7c and the PR description the `preinstall` script is not only getting run when the user is running `npm install` within the `ui-kit`  directory but also if you install the package as a dependency. If the `scripts` directory is not included in the distribution this command will fail which will fail the install. If we include the scripts directory in the distribution the git hook gets installed unnecessarily. I thought we might not want this. this is why I rather used a custom script.

Reply:
Let's just make it automated then so we don't have another step to remember/do when setting up this repository.